draft-ietf-rats-eat.md

```diff
@@ -283,10 +269,11 @@ Here are a few examples of claims:
 * Environmental characteristics of a device like its GPS location
 * Formal certifications received
 
-EAT also supports nesting of sets of claims and EAT tokens for use with complex composite devices.
+The EAT format supports nesting claims sets and EATs as well as the use of detached claim sets to enable representation of claims from use cases featuring complex composite devices. Owing to the goal of supporting a broad range of use cases, and because {{RFC8392}} and {{RFC7519}} are very flexible, the profile concept (see {{profiles}}) is defined to enable the usage of the EAT format to be tailored to specific use cases to ensure interoperability.
 
-This document uses the terminology and main operational model defined in [RATS.architecture].
-In particular, it can be used for evidence and attestation results.
+This document uses the terminology and operational models defined in [RATS.architecture].
+In particular, EATs can be used for evidence and attestation results.
```

Comment:
I would transplant this sentence at line 257.

```suggestion
In particular, EATs can be used for evidence and attestation results.
```

Reply:
Moved into the opening paragraph. Also moved the architecture reference to the second paragraph.